src/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <section id="todo-list">
+        <div id="todo-header">
+            <h4>Today's To do</h4>
+            <span class="material-icons">refresh</span>
+        </div>
+        <div class="input-task">
+            <input type="text" id="addTask" placeholder="Add to your list...">
+            <button type="button" id="addTaskAction" class="material-icons">check</button>
+        </div>
+        <ul id="todo-container">
+        </ul>
+        <button type="button" id="wipe-tasks">Clear all completed tasks</button>
+    </section>
+</body>
+</html>
```

Comment:
For readability, I believe it would be great to use two spaces of indentation instead of four, following [HTML best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).